security/pkg/server/ca/authenticate/kubeauth/kube_jwt.go

```diff
@@ -74,8 +74,6 @@ func (a *KubeJWTAuthenticator) AuthenticatorType() string {
 	return KubeJWTAuthenticatorType
 }
 
-const DefaultKubernetesAudience = "kubernetes.default.svc"
```

Comment:
should we just allow `kubernetes.default.svc` OR `kubernetes.default.svc.cluster.local`? 

I think that we should work out of the box with common environments (like https://github.com/istio/istio/issues/33744), and for bespoke cases they should use third-party-jwt. WDYT?

Reply:
Ok, fixed.